sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/aggregator/AggregatorHandle.java

```diff
@@ -104,4 +111,8 @@ protected void doRecordDouble(double value) {
     throw new UnsupportedOperationException(
         "This aggregator does not support recording double values.");
   }
+
+  public boolean isValuesRecorded() {
```

Comment:
How about?
```suggestion
  public boolean hasRecordedValues() {
```
Can you add javadoc?

Reply:
Sounds good. I'll change and add javadoc